src/Bridge/Symfony/Routing/IriConverter.php

```diff
@@ -120,8 +123,20 @@ public function getIriFromItem($item, int $referenceType = UrlGeneratorInterface
 
         try {
             $identifiers = $this->generateIdentifiersUrl($this->identifiersExtractor->getIdentifiersFromItem($item), $resourceClass);
+            $currentRouteParams = $this->request->attributes->get('_route_params');
+            $pathVariables = $route = $this->router->getRouteCollection()->get($routeName)->compile()->getPathVariables();
```

Comment:
This is very bad for performance.

Reply:
Yes you are right, is compiled route not cached?